### DISCUSSION TITLE
Ejecutar un programa de python

### DISCUSSION THREAD

**Discussion by AlanAcosta17** [LaunchX-InnovaccionVirtual/CursoIntroPython#72] [Q&A]
Buenas, puede que sea una pregunta algo tonta, estoy en la documentación del modulo 1 y estaba intentando ejecutar el programa que viene ahí (la suma sencilla), pero al momento de ejecutar ese llamado me marca un error. ¿Saben porque sea?
![M1](https://user-images.githubusercontent.com/99235417/153343450-762e0407-9b57-4d96-a310-69fc6ca00f65.PNG)

**Comment by LeonardoCelisH** [comment]
Hola, cuando trabajas con el notebook se ejecuta propiamente en la celda (como lo hiciste en la primera), en la parte de abajo te arroja el resultado, por tanto no hay necesidad de que pongas lo de la segunda celda (solo aplica cuando trabajas en consola si no me equivoco)

**Reply by AlanAcosta17** [reply]
Ah okay eso sería si lo estuviera escribiendo dentro de la terminal (cmd)

**Reply by rarmz** [reply]
Exacto

**Comment by ViktorVizcarra** [comment]


**Reply by ViktorVizcarra** [reply]
> > Buenas, puede que sea una pregunta algo tonta, estoy en la documentación del modulo 1 y estaba intentando ejecutar el programa que viene ahí (la suma sencilla), pero al momento de ejecutar ese llamado me marca un error. ¿Saben porque sea? ![M1](https://user-images.githubusercontent.com/99235417/153343450-762e0407-9b57-4d96-a310-69fc6ca00f65.PNG)
> 
> creo que el problema es que este codigo "python3 program.py" se debe hacer desde consola(cmd), lo cual le estas diciendo que ejecute el archivo program.py que es el programa que te resuelve la suma(el cual tiene el codigo que esta arriba, si borras python3 program.py y guardas ese notebook como tipo de archivo python, habras creado el programa que quieres ejecutar)...

**Reply by AlanAcosta17** [reply]
Gracias

**Comment by Dfredude** [comment]
Como dice el error de la consola es un "Syntax Error". El comentario `# program.py`  supongo que lo agregaron para indicar que dentro de la celda es como si tuvieras un archivo `.py` pero no existe el archivo como tal. Aun si existiera el archivo, el comando `Python3 program.py` solo se usaria para correlo cuando usas el CLI (cmd en windows); **_no dentro de un script de Python_**. Aqui estas usando **_VS Code_**, y solo necesitas presionar el boton **_Play_** o **_Play all_**.

**Reply by AlanAcosta17** [reply]
Muchas gracias :3

**Comment by RicardoUValencia** [comment]
Eso se hace en consola, si quieres que funcione como lo planteas desde de crear un archivo llamado program.py y dentro del archivo escribir todo el codigo y entonces ahora si desde la consola y ahi puedes entonces ejecutar python3 program.py
